tests/CMakeLists.txt

```diff
@@ -23,5 +23,6 @@ elseif(APPLE)
     endif()
 elseif(WINDOWS)
     message("WINDOWS!")
+    target_compile_options(tileson_tests PUBLIC /bigobj)
```

Comment:
The windows build on github actions was complaining about the /bigobj flag missing, that is why I added it. It does not seem to be required for local builds. 
As this only changes the compile options of the test, it does not seem critical.

Reply:
Sounds like a weird and unnecessary requirement by Windows, but if it solves an issue on Windows by adding this option, I'll allow it 😛